.circleci/config.yml

```diff
@@ -14,7 +14,7 @@ commands:
     description: "Setup and start fecfile-web-api for E2E tests"
     steps:
       - setup_remote_docker:
-          docker_layer_caching: false
+          docker_layer_caching: true
```

Comment:
I believe this was false for a reason.  I think we want the api to rebuild in case there were changes to it.  Those changes wouldn't trigger a change to the docker file, which is what validates/invalidate the docker_layer_cache